### PR TITLE
master

### PR DESCRIPTION
**Pull request by NickHeidinger**
fixed alignment, added form highlights, added slack link, header background needs work still

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -214,11 +211,11 @@ <h4 ><span>Wynand</span> Badenhorst</h4>
         <!-- Co-VP of Outreach -->
         <div class="col-md-3">
                 <div class="exec_img">
-                    <img src="images/harman.jpg" alt="Harmon" />
+                    <img src="images/harman.jpg" alt="Harman" />
+                    <span class="title">Co-VP of Outreach</span>
```
Comment (asikkenga) [inline index.html]: Why did you move this up?  I only see the change on Harman's

Comment (NickHeidinger) [reply index.html]: Moved up? I fixed a typeo in his name

Comment (asikkenga) [reply index.html]: The span is moved inside the div. It belongs outside. Otherwise nice catch on the spelling

**Comment by NickHeidinger** [comment]
projects-page

File: index.html (hunk 1 above)
Comment (NickHeidinger) [reply index.html]: oh i dont think it affects the page rendering

Comment (NickHeidinger) [reply index.html]: fixed*
